.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+Please refer to the [faqs](http://openkore.com/index.php/Frequently_Asked_Questions) (such as ["Packet Tokenizer"](http://openkore.com/index.php/Frequently_Asked_Questions#Packet_Tokenizer:_Unknown_switch:_xxxx)), [forum](http://forums.openkore.com), or [documentation wiki](http://openkore.com/index.php/Main_Page) for help using OpenKore.
+
+* Before submitting an issue, ensure you are on the latest release
+* Check if an issue already exists for your request
+* Please post in English
```

Comment:
We could probably be more conversational here. Maybe something like "We know English might not be your first language; it certainly isn't for many of us too! But please do try to post in English.